src/BUILDING.md

```diff
@@ -17,7 +22,7 @@
 * C++11 compiler *(see platform specific requirements)*
 
 ### Optional Tools
-* [Doxygen 8.11](http://www.stack.nl/~dimitri/doxygen/download.html) *(If building source documentation)*
+* [Doxygen 8.11](http://www.stack.nl/~dimitri/doxygen/download.html) *(for building source documentation)*
```

Comment:
I found that doxygen was required in order to run `cmake ../src`.  If it wasn't installed cmake died.